tests/CMakeLists.txt

```diff
@@ -233,20 +236,20 @@ if(WITH_PYTHON)
     TEST_FILES restarting/from_7.0.0/UpgradeAndBackupRestore-1.toml
                restarting/from_7.0.0/UpgradeAndBackupRestore-2.toml)
   add_fdb_test(
-    TEST_FILES restarting/to_7.0.0/CycleTestRestart-1.toml
-               restarting/to_7.0.0/CycleTestRestart-2.toml)
+    TEST_FILES restarting/to_7.1.0/CycleTestRestart-1.txt
+               restarting/to_7.1.0/CycleTestRestart-2.txt)
```

Comment:
I think I understand why this one needs to be txt, but why do the ones below not work? It seems like they should only be running for versions that understand toml.

I would also imagine that we could leave the the first test file from pre-upgrade in txt and leave the second one upgraded.

Reply:
Yeah, this one works, but there's a bug uncovered.
Just want to disable it first while fixing it to unblock others.

Reply:
I added a comment here.
There are several restarting bugs uncovered after we picked up the 7.0 binary.